.travis.yml

```diff
@@ -8,3 +8,4 @@ before_install: gem update bundler
 
 script:
   - rake test
+  - codeclimate-test-reporter
```

Comment:
The [docs](https://docs.codeclimate.com/v1.0/docs/travis-ci-ruby-test-coverage) mention to prepend this line with `bundle exec`, but it appears to be working for now, so I guess it's OK